pom.xml

```diff
@@ -113,9 +113,15 @@
             <version>1.7R5</version>
         </dependency>
         <dependency>
-          <groupId>com.jayway.jsonpath</groupId>
-          <artifactId>json-path</artifactId>
-          <version>2.0.0</version>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
+            <version>2.0.0</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>net.minidev</groupId>
+                    <artifactId>asm</artifactId>
```

Comment:
@gianm I think if we upgrade to JSONPath 2.1.0, we can exclude json-smart completely:

https://github.com/jayway/JsonPath/pull/128

This PR is present in 2.1.0 and it skips the creation of the DefaultsImpl object if both a JsonProvider and MappingProvider are given to the builder before build() is called